api/v1alpha1/releaselink_webhook.go

```diff
@@ -36,12 +41,11 @@ var _ webhook.Defaulter = &ReleaseLink{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (r *ReleaseLink) Default() {
-	label := "release.appstudio.openshift.io/auto-release"
-	if _, found := r.GetLabels()[label]; !found {
+	if _, found := r.GetLabels()[autoReleaseLabel]; !found {
 		if r.Labels == nil {
 			r.Labels = make(map[string]string)
 		}
-		r.Labels[label] = "true"
+		r.Labels[autoReleaseLabel] = "true"
```

Comment:
This looks fine, but I have a trivial question.
Looks like `r.GetLabels()` would potentially not succeed if `r.Labels == nil`.
So I feel like maybe one would want to check if the pointer is `nil` first, then check if the value is empty?
So I'm wondering if this ordering is more optimal, less complex, or does less work? 

Also, I wonder if the `r.Labels = make(map[string]string)` could instead be a [map literal](https://go.dev/ref/spec#Composite_literals) instead?
Something like:
```
r.Labels = map[string]string{autoReleaseLabel : "true"}
```
Note: this is not asking for code changes, feel free to ignore.

Reply:
Line 45 already checks for nil, but I did make the change to set the label when the labels are created instead of in another line. Thanks